test/Microsoft.ML.Tests/Scenarios/IrisPlantClassificationWithStringLabelTests.cs

```diff
@@ -30,6 +30,14 @@ public void TrainAndPredictIrisModelWithStringLabelTest()
             pipeline.Add(new StochasticDualCoordinateAscentClassifier());
 
             PredictionModel<IrisDataWithStringLabel, IrisPrediction> model = pipeline.Train<IrisDataWithStringLabel, IrisPrediction>();
+            string[] scoreLabels;
+            model.TryGetScoreLabelMapping(out scoreLabels);
```

Comment:
>model.TryGetScoreLabelMapping(out scoreLabels); [](start = 12, length = 47)

Why I can't just define 
public class IrisPrediction
        {
            [ColumnName("Score")]
            public float[] PredictedScores;

            [ColumnName("OriginalLabels")]
            public string[] OriginalLabels;
        }
and fill it automatically if it's possible?

Reply:
`IrisPrediction` is a per-row structure. This is metadata, which is a property of the schema itself.

---
In reply to: [190754719](https://github.com/dotnet/machinelearning/pull/239#discussion_r190754719) [](ancestors = 190754719)